src/app/nft-drop-mgr/nft-drop-mgr.component.html

```diff
@@ -97,8 +97,9 @@
 <simple-center-loader *ngIf="loading"></simple-center-loader>
 
 <div class="w-100 light-grey-divider border-bottom border-top border-color-grey" style="height:10px"></div>
-
-<div *ngIf="posts.length > 0 && (!loading || loadingNewDrop)" class="p-15px fs-15px">
+<tab-selector [tabs]="feedTabs" [activeTab]="activeTab" (tabClick)="_handleTabClick($event)"></tab-selector>
+<!--Admin panel-->
+<div *ngIf="showShowcaseManagementTab()" class="p-15px fs-15px">
   <b *ngIf="!loadingNewDrop">NFTs in this drop:</b>
   <div #uiScroll *uiScroll="let post of datasource" [ngClass]="{'d-flex align-items-center mt-15px border border-color-grey br-8px': !loadingNewDrop}">
```

Comment:
this uiScroll directive needs to be moved into a separate component. If you switch back and forth between the tabs, you're going to lose the Showcase Management view. You'll see an error that looks like the below:

```
ERROR TypeError: Cannot read property 'dispose' of undefined
    at UiScrollComponent.ngOnDestroy (ngx-ui-scroll.js:57)
    at executeOnDestroys (core.js:7405)
    at cleanUpView (core.js:7318)
    at destroyViewTree (core.js:7144)
    at destroyLView (core.js:7296)
    at ViewContainerRef.remove (core.js:23208)
    at ViewContainerRef.clear (core.js:23116)
    at NgIf._updateView (common.js:3535)
    at NgIf.set ngIf [as ngIf] (common.js:3502)
    at setInputsForProperty (core.js:10961)
```